src/static/css/cf-enhancements.less

```diff
@@ -972,6 +972,10 @@ textarea.input__long {
 .content__half-top-on-desk {
     .respond-to-min(800px, {
         padding-top: 20px;
+
+        &:after {
+          top: 20px!important;
```

Comment:
Re: the sidenote: Fairly sure we had to do it that way for some reason. Feel free to try it the other way and see if it causes problems.